lib/task-data/schemas/types-installos.json

```diff
@@ -319,6 +324,14 @@
             "type": "string",
             "description": "Extra (persistent) kernel boot parameters",
             "minLength": 1
+        },
+        "Timezone": {
+            "description": "The Timezone based on $TZ (https://en.wikipedia.org/wiki/List_of_tz_database_time_zones)",
+            "type": "string"
+        },
+        "Ntp": {
+            "description": "The NTP server address",
+            "type": "string"
```

Comment:
We could add `minLength` for both `Timezone `and `Ntp`.

Reply:
if i set minlength it means its a required option or not ?

Reply:
No, whether an options is required or not is defined in `install-debian.json`.